cmd/thanos/rule.go

```diff
@@ -69,6 +69,12 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application, name string)
 
 	evalInterval := cmd.Flag("eval-interval", "the default evaluation interval to use").
 		Default("30s").Duration()
+	tsdbMaxBlockDuration := cmd.Flag("tsdb.max-block-duration", "maximum duration for TSDB Block. Suggested to use the same value as minimum to disable dynamic resizing").
+		Default("2h").Duration()
+	tsdbMinBlockDuration := cmd.Flag("tsdb.min-block-duration", "minimum duration for TSDB Block. Suggested to use the same value as maximum to disable dynamic resizing").
+		Default("2h").Duration()
```

Comment:
When running Thanos, it is a safety requirement for Prometheus to have those two flags being set to the same value. Thus I'm not sure it makes sense to expose both here to begin with.

I'd just kill the max block duration options.

Reply:
k, makes sense